content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,199 @@
+# List Your Exchange on XRP Charts
+
+In addition to providing data about the XRP Ledger network and its accounts and transactions, XRP Charts also provides [XRP market data](https://xrpcharts.ripple.com/#/xrp-markets) from external exchanges. This tutorial describes how to have your exchange and its XRP trade and order book data listed on XRP Charts.
+
+To enable XRP Charts to list your exchange, complete the following tasks:
```

Comment:
Some (most?) exchanges probably have some version of these endpoints (for example, Kraken has one for [recent trades](https://www.kraken.com/help/api#get-recent-trades) and the [order book](https://www.kraken.com/help/api#get-recent-trades)). I wonder if we should add some language that makes it clear that the exchange doesn't necessarily have to take any action if they already provide these endpoints. Like, we need these endpoints and this data and would prefer these formats, etc, but if you have something close, you (exchange) might not need to do anything.

Reply:
Great point! Thank you.

Reply:
I agree with @ryangyoung - we could add some clarity on these endpoints , so that they don't have any duplicate effort on their end .

Reply:
I think it might be useful to break this up into "You'll need to make the following data available, if you don't already" and "Then send a listing request to XRP Charts". The timeline on those two steps is potentially pretty different so it makes sense to split them up, and that also makes it clearer from the get-go that exchanges may not need to implement new stuff from scratch.